sdk/ml/azure-ai-ml/azure/ai/ml/_schema/component/parallel_component.py

```diff
@@ -35,6 +35,11 @@ class ParallelComponentSchema(ComponentSchema):
     mini_batch_size = fields.Str(
         metadata={"description": "The The batch size of current job."},
     )
+    partition_keys = fields.List(
+        fields.Str(),
```

Comment:
you may add the logic in a post_load function here

Reply:
this part should be refactored in a dedicated Pull request.